nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -76,17 +100,20 @@ in
         };
       };
     };
-
-    secretFile = mkOption {
-      type = with types; nullOr path;
-      default = null;
-      description = lib.mdDoc "Path to a secret JSON configuration file which is merged at runtime with the one generated from {option}`services.lemmy.settings`.";
-    };
   };
 
   config =
+    let
+      secretOptions = {
+        "pictrsApiKeyFile" = { setting = [ "pictrs" "api_key" ]; path = cfg.pictrsApiKeyFile; };
```

Comment:
Why do you quote them?

Reply:
Good question, I think I was in Python mode.